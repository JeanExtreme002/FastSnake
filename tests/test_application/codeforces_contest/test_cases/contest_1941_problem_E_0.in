5
3 11 1 4
0 1 2 3 4 5 4 3 2 1 0
0 1 2 3 2 1 2 3 3 2 0
0 1 2 3 5 5 5 5 5 2 0
4 4 2 1
0 3 3 0
0 2 1 0
0 1 2 0
0 3 3 0
4 5 2 5
0 1 1 1 0
0 2 2 2 0
0 2 1 1 0
0 3 2 1 0
1 8 1 1
0 10 4 8 4 4 2 0
4 5 3 2
0 8 4 4 0
0 3 4 8 0
0 8 1 10 0
0 10 1 5 0
