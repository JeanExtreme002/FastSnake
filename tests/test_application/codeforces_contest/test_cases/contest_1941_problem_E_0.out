4
8
4
15
14
